src/PrestaShopBundle/Form/Admin/Sell/Product/Specification/FeaturesType.php

```diff
@@ -33,9 +33,29 @@
 use Symfony\Component\Form\Extension\Core\Type\CollectionType;
 use Symfony\Component\Form\FormBuilderInterface;
 use Symfony\Component\OptionsResolver\OptionsResolver;
+use Symfony\Component\Translation\TranslatorInterface;
 
 class FeaturesType extends TranslatorAwareType
 {
+    /**
+     * @var bool
+     */
+    protected $isFeatureEnabled;
+
+    /**
+     * @param TranslatorInterface $translator
+     * @param array $locales
+     * @param bool $isFeatureEnabled
+     */
+    public function __construct(
+        TranslatorInterface $translator,
+        array $locales,
+        bool $isFeatureEnabled
```

Comment:
I think this is a BC break ;)

Reply:
@atomiix : I check the ADR : `Since classes whose instantiation is delegated to the Symfony service container are not supposed to be instantiated individually, changing the constructor of those classes is allowed.`

So it should not be a BC Break.

Reply:
You're right! I forgot we had this rule!